webapp/components/drawer/Drawer.js

```diff
@@ -58,17 +59,50 @@ const DrawerItem = ({
   </Tooltip>
 );
 
-export default ({
+
+const drawerStyles = ({spacing}) => ({
```

Comment:
We have now two different ways of styling for one component here: withStyles (Higher order component) and useStyles (Hook). Don't know if we want to use useStyles for functional components and withStyles for class components or rather only withStyles for all, but I would stick to one approach per component

Based on doc (https://material-ui.com/styles/basics/#hook-api) they both work in the same way, useStyles was just introduced to match the functional components pattern